javascript/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>arrow.js browser test</title>
+    <meta charset="utf-8">
+    <style>
+/* adapted from http://codepen.io/brennowilliam/pen/PwJoPq */
```

Comment:
I am happy to review more closely on GitHub, but only after resolving the IP lineage problem. It seems like this link only involves CSS, is there anything else? If that's it then I recommend removing the offending code or replacing it with something with a known ASL 2.0-compatible license